app/javascript/pages/browse/TileListItem.vue

```diff
@@ -9,15 +9,16 @@
           <b-icon v-if="showUrgentIcon" icon="exclamation-triangle" size="is-small" />
           {{ urgency.name }}
         </b-tag>
+        <MappedIconList :contactTypes="[{id: contribution_type, name: contribution_type}]" />
```

Comment:
this is contactTypes, but using contribution_type data?

Reply:
Good point — I'll update this param name